.mage/gen.mk

```diff
@@ -3,6 +3,8 @@ mage_targets_path := $(mage_folder)/gen
 mage_targets := $(mage_targets_path)/targets.mk
 mage := $(mage_targets_path)/local_mage
 
+include $(mage_targets)
```

Comment:
Is there any reason we have `mage_targets` and `mg_targets`? They are hard to tell apart by their name

Reply:
Yeah I agree, i struggled a bit with what I should call it, so happy to change. `mage_targets` contains the variables that points to the actual makefiles that holds the targets. `mg_targets` is the default one which holds all targets that are not namespaced

Reply:
if one contains files and the other contains targets, then how about `mage_files` and `mage_targets`?

Reply:
or `mage_target_files`?

Reply:
or, if `mage_targets` is the *one magic variable* to always include - maybe we can elevate it and call it `mage_tools` and then to use mage-tools you need to depend on the top-level `mage_tools` variable?

Reply:
I like `mage_files`. The reason i went for `mg_targets` was mainly because i went with `mg_terraform` etc. But we could go for `mage_terraform`

The variable in `gen.mk` will only be included there.

The root makefile looks something like:
```
.PHONY: all
all: $(mg_targets)

include .mage/gen.mk
include $(mg_targets)
```

So `mage_tools` dsnt really fit it imo. But we could go for your first suggestion?